tests/testthat/_snaps/tidy_as_ard.md

```diff
@@ -96,3 +96,40 @@
       14 Planned unit testing error!
       15 Planned unit testing error!
 
+---
+
+    Code
```

Comment:
Is it possible to abbreviate the data frame that is being snapshot to not print in multiple chunks? It makes is hard to track any changes in the future.